ui/apps/platform/src/Containers/SystemConfig/SystemConfigPage.tsx

```diff
@@ -87,6 +96,7 @@ const SystemConfigPage = (): ReactElement => {
                     setSystemConfig={setSystemConfig}
                     setIsNotEditing={setIsNotEditing}
                     isCustomizingPlatformComponentsEnabled={isCustomizingPlatformComponentsEnabled}
+                    defaultRedHatLayeredProductsRule={defaultRedHatLayeredProductsRule || ''}
```

Comment:
Nit: If the request to get this data fails with an error, we will still render the component with the empty string right?

I don't think this is a blocker, but it would be nice to have some indication that the request failed.